src/lib/components/tabs/Tabs.component.js

```diff
@@ -33,13 +34,14 @@ const TabItem = styled.div`
   flex-basis: 15%;
   flex-shrink: 1;
   text-align: center;
-  ${props => {
+  ${(props) => {
     return (
       !props.selected &&
       css`
         &:hover {
           cursor: pointer;
-          background-color: ${getTheme(props).primary};
+          background-color: ${props.activeTabColor ||
+          getTheme(props).primaryDark1};
```

Comment:
Why don't we keep `primary` for our default background color? This change will most likely break scality/zenko-ui.

Reply:
ok. thanks for the review!

I changed back to `primary` as the default background color.
I will set `activeTabColor` in my case.